modules/serverless-workflow-tag-revisions-assign.adoc

```diff
@@ -3,42 +3,34 @@
 // serverless/knative-client.adoc
 
 [id="assigning-tag-revisions_{context}"]
-= Assigning tag revisions 
+= Assigning tag revisions
 
-A tag in a traffic block of service creates a custom URL, which points to a referenced revision. A user can define a unique tag for an available revision of a service which creates a custom URL by using the format `http(s)://TAG-SERVICE.DOMAIN`.
+A tag in a traffic block of a service creates a custom URL, which points to a referenced revision.
+A user can define a unique tag for an available revision of a service which creates a custom URL by using the format `http(s)://TAG-SERVICE.DOMAIN`.
 
-A given tag must be unique to its traffic block of the service. `kn` supports assigning and unassigning custom tags for revisions of services as part of the `kn service update` command.
+A given tag must be unique to its traffic block of the service.
+`kn` supports assigning and unassigning custom tags for revisions of services as part of the `kn service update` command.
 
 [NOTE]
 ====
 If you have assigned a tag to a particular revision, a user can reference the revision by its tag in the `--traffic` flag as `--traffic Tag=Percent`.
 ====
 
-
 .Procedure
-* Use the following command:
+
+* Assign tag revisions by updating the service:
 +
+[source,terminal]
 ----
 $ kn service update svc --tag @latest=candidate --tag svc-vwxyz=current
 ----
-
-[NOTE]
-====
++
 `--tag RevisionName=Tag` uses the following syntax:
 
-* `--tag` flag requires two values separated by a `=`.
-
-* `RevisionName` string refers to name of the `Revision`.
-
-* `Tag` string denotes the custom tag to be given for this Revision.
-
-* Use the identifier `@latest` for the RevisionName to refer to the latest ready revision of the service. You can use this identifier only once with the `--tag` flag.
-
-* If the `service update` command is updating the configuration values for the Service (along with tag flags), `@latest` reference will be pointed to the created Revision after applying the update.
-
-* `--tag` flag can be specified multiple times.
-
-* `--tag` flag may assign different tags to the same revision.
-====
-
-
+** `--tag` flag requires two values separated by a `=`.
+** `RevisionName` string refers to name of the `Revision`.
+** `Tag` string denotes the custom tag to be given for this Revision.
+** Use the identifier `@latest` for the RevisionName to refer to the latest ready revision of the service. You can use this identifier only once with the `--tag` flag.
+** If the `service update` command is updating the configuration values for the Service (along with tag flags), `@latest` reference will be pointed to the created Revision after applying the update.
```

Comment:
s/the Service (along with tag flags), `@latest` reference will be pointed t0/the service, along with tag flags, `@latest` reference is pointed to